src/pages/PostComments.jsx

```diff
@@ -0,0 +1,39 @@
+import { useContext, useEffect, useState } from 'react';
+import { keys } from '../network/keys';
+import { urls } from '../network/urls';
+import { useGetHandler } from '../network/useQueryClient';
+import { filterComments } from '../utils/utils';
+import { CircularProgress } from '@mui/material';
+import { context } from '../context/ContextProvider';
+import Comments from '../components/Comments';
+import CommentHeader from '../components/CommentHeader';
+
+const PostComments = () => {
+  const [comments, setComments] = useState([]);
+  const { postId, post } = useContext(context);
```

Comment:
instead of context get id of post from url and then fetch it and comments

Reply:
First fetch post and then fetch comment from API, learn about sequential API calling